source/manuals/security-overview-for-websites.html.md.erb

```diff
@@ -0,0 +1,249 @@
+---
+title: Security overview for websites
+last_reviewed_on: 2021-06-25
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+These 15 steps are primarily a for running independent websites that are not on the gov.uk domain; however, the principles here apply to all Cabinet Office websites and applications.
+
+## 1. Access Control
+**Must Do**
+
+1. Use named accounts to log in for all systems (from registrars to content
+management systems)
+  - Ideally linked to Google using SSO (Single Sign-On) - like
+OAuth or OIDC.
+  - If you can’t use SSO, make sure you use a gov.uk email and enable MFA
+(Multi-Factor Authentication) and ideally using a phishing-resistant mechanism
+like WebAuthn (hardware keys).
+2. Have a process to review who has access regularly.
+3. If available, enable different roles for different people’s needs.
+4. See [how to manage access to your third-party service accounts](/standards/accounts-with-third-parties.html) for more details.
+
+**Why?**
+
+Using named accounts makes incident response and non-repudiability (make it harder for someone to deny it was them) easier because if an incident occurs, it is easier to work out the cause or vector to restrict access.
+
+Using SSO makes JML (Joiners, Movers and Leavers) processes simpler. As soon as someone leaves and their Google/IT account is removed, the access to services will be removed.
+
+See the [NCSC's guidance on identity and access management](https://www.ncsc.gov.uk/guidance/introduction-identity-and-access-management).
+
+## 2. Domain Name Registrar
+**Must Do**
+
+Make it more challenging for adversaries to take over your registrar, which is where your domain is configured.
+
+Along with [Access Control], do this by:
+
+1. Contacting your registrar and ask if they offer any additional safeguards to prevent take-over.
+2. Asking your registrar for contact details that you can use in the event of an incident. Note these in your incident response plan.
+3. If available, enabling DNSSEC ([Cloudflare has a great explanation on DNSSEC](https://www.cloudflare.com/en-gb/dns/dnssec/how-dnssec-works/))
+4. Ensure your whois details are correct and contain the organisation or department name, the main building address, and a generic shared email address (_service_-webadmin@organisation[.]gov.uk) and don’t enable privacy guard.
+
+**Why?**
+
+If an adversary successfully gets access, they could redirect your domain to a malicious or damaging website, and it might be difficult to revert.
+
+Having correct and available whois records helps malware and security researchers understand the domain’s ownership and reach out if there’s an issue. It can also help prevent search engines from delisting or flagging the domain as spam.
```

Comment:
```suggestion
Having correct and available WHOIS records helps malware and security researchers understand the domain’s ownership and reach out if there’s an issue. It can also help prevent search engines from delisting or flagging the domain as spam.
```